### DISCUSSION TITLE
Equipo 6 Proyecto Final

### DISCUSSION THREAD

**Discussion by KasturiSShah** [PatronesHermosos/design-thinking-1ra-generacion#12] [Proyecto Final]
Hola @Gaga1982 
Aquí está el hilo en el foro para tu equipo. Podrías conversar aquí sobre tus avances con el proyecto y avisar el Equipo de Educación de Patrones Hermosos sobre cualquier pregunta que tengas al respecto.
Con la emoción de seguir el desarrollo de tu proyecto,
Kasturi

**Comment by Gaga1982** [comment]
Hola recien llego de trabajar y vi tu mail. Me sigue dando error. Error algo salio mal dice.Ni siquiera puedo abrir tu archivo para leerlo y mandartelo x mail

**Reply by KasturiSShah** [reply]
Hola Gabriela,
No te preocupes. ¿Podrías contestar al correo que te mandé con tu entrega? ¿O no está funcionando tu correo tampoco?
Pronto resolveremos este asunto
Kasturi

**Reply by Gaga1982** [reply]
Necesito que me mandes la consigna o se lo que esta escrito en el readme para contestar las preguntas de vuelta y mandarlas por el mail por favor

**Reply by Gaga1982** [reply]
Porque ni siquiera puedo abrir eso me sale ese error

**Reply by KasturiSShah** [reply]
Ah, que pena. Claro que sí, está todo a continuación:

--

Etapa 0 del Curso Design Thinking 1ra Generación de Patrones Hermosos
¡Bienvenid@s!

Para completar la etapa 0 del proyecto final, llena la siguiente información:

### Información General
Nombre del equipo:

Nombre de los miembros del equipo:

Describa en una sola frase el problemática relevante para tu comunidad (el foco de tu proyecto final):

### Protocolo de la entrevista:
3-5 preguntas que harás durante tu entrevista: (Para construir preguntas apropiadas, no dudes en consultar los vídeos en la plataforma y las diapositivas desde la sesión interactiva)

1.

2.

3.

Entendimos que el protocolo puede cambiar. El proposito principal del protocolo en esta etapa es prepararte para hacer una entrevista la semana que viene con un miembro de la comunidad que tu proyecto puede servir (o, en otros términos, un posible usuario de tu producto). Por eso, esta semana, también es importante identificar la persona que entrevistarás.

**Comment by Gaga1982** [comment]
![Captura de pantalla (1)](https://user-images.githubusercontent.com/100386928/223578829-b6993c94-c5d1-4e9d-9657-3b12f56fbee6.png)
no se si ahi te sale la captura

**Reply by KasturiSShah** [reply]
Gracias por la captura @Gaga1982. Recibida. 
¿Podrías intentar abrir esta liga? https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-06/etapa-0/README.md

Si no te sale, ¿podrías enviarme la información de la etapa 0 por correo?

**Reply by Gaga1982** [reply]
Ya lo intente y me sigue dando error. No se que pasa con mi Git Hub y si abro una cuenta nueva? Ya te lo mande por e-mail. Perdón por los inconvenientes causados y muchas gracias por la espera. Siempre dando la nota yo

**Reply by KasturiSShah** [reply]
Hola @Gaga1982 
Gracias por la entrega recibida por correo. He puesto todo en tu archivo de Github: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-06/etapa-0/README.md
Y no es así, no te preocupes. No tiene nada que ver contigo, creo que hay algo raro pasando con la cuenta. En la asesoría después de revisar las mapas de sintetización, revisaremos este asunto.
Saludos,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @Gaga1982 

Gracias por entregar la etapa 0 de tu proyecto final. Estoy revisándola y te escribo con retroalimentación. En primer lugar, me parece muy importante el tema que escogiste (¡y relevante también para la misión de Patrones Hermosos! Si quieres charlar sobre los retos y las oportunidades en esta área, me avisas, sería un gusto aprender juntas.)

Tengo algunas sugerencias. Las preguntas son relevantes, y me interesa especialmente el #4 y #5. Sobre el #5, ¿hay manera de escribirla más generalmente para dar espacio para el entrevistado/a a tener diferente opinion? Por ejemplo, _"Por último ¿usted considera la educación como una prioridad transversal que trasciende las distintas gestiones de gobierno? O, ¿Considera que hay limitaciones con esta perspectiva?"_

Sugiero agregar algo para romper el hielo (pregunta, historia, broma, etc). También, avisar al entrevistador/a sobre la duración de la entrevista _antes_ de empezar, para que esté preparado. Y, ¿qué opinas de agregar una pregunta sobre la motivación del entrevistado/a a trabajar en el trabajo que tiene?

¿A quién vas a entrevistas? Según las preguntas, parece que sería alguien que trabaja dentro de la educación tecnológica, pero si no es así, sería importante checar que cada pregunta sea relevante para el entrevistado/a. Sugiero agendar y completar la entrevista tan pronto como sea posible, para que tengas tiempo suficiente para crear la mapa para sintetizar tus apuntes.

 ¿Tienes preguntas para mí? Cualquier cosa, aquí ando.

¡Suerte!
Saludos,
Kasturi

**Reply by Gaga1982** [reply]
La entrevista es al intendente (alcalde) de la ciudad que es candidato a gobernador y la pregunta 5 no es intimidante porque justamente la idea es que como la educación tecnológica se está dando en el ambito municipal si gana las elecciones podría ampliarlo al provincial. Y lo puse en la introducción a las preguntas en el mail, lo mande completo pero no está en lo que subiste a mi repositorio de la carpeta 6. Pregunta ¿va a haber otra asesoría está semana? No puede asistir a la de anoche aquí eran las 22 hs y ya era tarde

**Reply by Gaga1982** [reply]
Informaciones Generales
Nombre del equipo: Quality Education Córdoba

Nombre de los miembros del equipo: Gabriela García Sánchez

Describe en una sola frase el problema relevante para tu comunidad (el foco de tu proyecto final):

Lograr que la educación tecnológica (programación y robótica)  llegue a todos los espacios de educación pública.

Protocolo de la entrevista:
3-5 preguntas que harás durante tu entrevista: (Para construir preguntas apropiadas, no dudes en consultar los vídeos en la plataforma y las diapositivas desde la sesión interactiva)

El entrevistado (ideal) es el intendente de la ciudad de Córdoba Martín Llaryora que es candidato a gobernador y ha implementado en la currícula de las Escuelas Municipales Programación y robótica y la entrevista se dirige a si se puede hacer lo mismo en las Escuelas Públicas provinciales.



¿ Cómo se está dando la implementación de programación y robótica en las escuelas municipales?
¿Usted cree que se puede hacer extensivo el proyecto a las escuelas públicas provinciales?
¿Los docentes de nivel inicial, primario y secundario reciben capacitaciones para acercar los contenidos a los niños o son docentes especializados en estas carreras los que dictan los contenidos?
¿Que opina de estas carreras tecnológicas?¿Lo considera el futuro?
Por último¿Usted considera la educación como una prioridad transversal que trasciende las distintas gestiones de gobierno?

**Reply by Gaga1982** [reply]
Aparte cuando los políticos están en campaña te vende lotes en la luna ni le va a resultar intimidante y lo más probable es que me conteste todo color de rosa 🤣🤣🤣 pero hablando en serio creo que si realmente le interesa el tema esto de la educación tecnológica publica desde la primera infancia puede pasar de lo micro a lo macro y llegar incluso a nivel nacional. Seré utópica pero mantengo la esperanza que sea accesible para todos y todas. Feliz día mujeres 💪🏻🌹

**Reply by KasturiSShah** [reply]
Gracias Gabriela por dar detalles sobre el entrevistado, y perdón por no haber visto el párrafo antes. 
Sobre la pregunta 5, podrías avanzar ahí como quieras dado que sabes mejor el contexto por ahí.
Suena muy interesante entrevistar a un político, ¡mucha suerte!
Habra una asesoría a las 10:15 AM (hora CDMX) este sábado. Una gran disculpa, me di cuenta que ayer la asesoría fue súper tarde para las que están en Argentina. La semana que viene, agendaremos una asesoría más temprano.
Cualquier cosa, aquí ando
Igualmente....feliz día!

**Comment by Gaga1982** [comment]
Buen día chicas. Espero que se pueda leer la carpeta 1. Gracias y buen inicio de semana

**Reply by KasturiSShah** [reply]
Hola @Gaga1982 
No puedo ver los cambios. ¿Quizás editaste la copia del repositorio que tienes?
Aquí está la liga que necesitas editar: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/tree/main/equipo-06/etapa-1
Igualmente, buen inicio de semana!
Kasturi

**Reply by Gaga1982** [reply]
https://github.com/PatronesHermosos/design-thinking-1ra-generacion

Quise hacer como me explicaste y no me deja, cuando toco el lápiz de editar sale error algo salió mal

**Reply by Gaga1982** [reply]
![Captura de pantalla (3)](https://user-images.githubusercontent.com/100386928/224733038-950793b6-bdec-42d8-abcf-473b0580d601.png)
![Captura de pantalla (4)](https://user-images.githubusercontent.com/100386928/224733073-aa9762c4-3bd1-4934-b3b8-ca65d0734d22.png)

**Reply by KasturiSShah** [reply]
Hola @Gaga1982 
Gracias por las capturas. Ahora entiendo. Parece que entregaste un comentario. Aquí está: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/commit/39bb475adc918348ca6b639e664e45fc3276d3f4
Usaré el comentario para compartir retroalimentación contigo mañana.
Y perdón, la liga que quería mandarte es esa: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-06/etapa-1/README.md
Saludos,
Kasturi

**Reply by Gaga1982** [reply]
Es que lo hice en un comentario porque no me deja de la forma correcta 😭

**Reply by KasturiSShah** [reply]
No te preocupes @Gaga1982, resolvimos todo y ahora está funcionando 😄 
Revisaré la etapa 1 pronto, y te escribiré con retroalimentación por este medio.
Saludos!

**Comment by KasturiSShah** [comment]
Hola @Gaga1982 

Gracias por entregar la etapa 1. La he revisado y escribo con algunas observaciones y sugerencias. En primer lugar, felicidades por haber completado la entrevista. ¡Parece que fue una entrevista muy interesante!

Mirando a la mapa, me parece OK crearla según las preguntas. Mi recomendación sería crear una mapa de afinidad según los temas principales cubiertos durante la entrevista. Los temas que veo son:

1. La estructura y la implementación el proyecto PIENSA (Apuntes en azul, verde, amarillo)
2. Papel de niños y educación en la sociedad (puntos en naranja y rosa)

¿Qué opinas de crear una mapa según estos dos temas? Solo para practicar, sé que es más fácil crear mapas con más apuntes/entrevistas, pero dado que estamos en un curso, sería buena idea aplicar los conceptos enseñados en las sesiones en vivo y en los vídeos.

Para la etapa 2, una sugerencia de mi parte sería hacer la actividad 8S (Ocho Loco) individualmente y luego usar SCAMPER (u otra estrategias en los videos) para ordenar las ideas y elegir una sola idea con que trabajarán en la etapa 3. (Es decir, repetir el proceso que practicamos durante las sesiones interactivas esta semana.)

¿Tienes preguntas para mí?

Mucha suerte,
Kasturi

**Reply by Gaga1982** [reply]
Hola Kasturi. Me gustaría por favor que me mandaras el enlace de la etapa 2 así puedo subir el trabajo en el repositorio porque tengo el mismo inconveniente que con los dos anteriores. Muchas gracias :smile:

**Reply by KasturiSShah** [reply]
Hola @Gaga1982 buenos días,
Claro que sí, aquí tienes: https://github.com/PatronesHermosos/design-thinking-1ra-generacion/blob/main/equipo-06/etapa-2/README.md
Me avisas como andas,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @Gaga1982 

Espero estés bien. Gracias por entregar la etapa 2 del proyecto final. La he revisado, y te escribo por este medio con retroalimentación. En primer lugar, gracias por organizar su 8S en términos de "PIEnsA" y educación de los niños en la sociedad. La organización me apoya entender la lógica que estás usando. 

Tengo varias preguntas:
- ¿Con cual (o cuales) ideas trabajarás para crear tu prototipado low fidelity? Mi recomendación sería elegir 1-2 ideas con que trabajarás. Si quisieras, podrías usar SCAMPER para reducir y organizar las 6 ideas en los Post-Its que pusiste en la entrega 2.
- No veo la liga a tu video grabado para la entrega 2. Me avisas si encuentres problemas al insertar la liga del video en el archivo de Github.
- ¿Cuál sería el prototipado low fidelity que crearás para la entrega 3?
- ¿Cómo andas con la entrega 3? Si necesitas una prórroga, me avisas.

Finalmente, ¿sería posible discutir tu idea, quizás en una asesoría? Me encantaría hacerlo. Veo muchos alineamientos con la iniciativa Patrones Hermosos y quizás podríamos colaborar/complementar. 

¿Tienes preguntas para mí? No hay prisa para contestar a este mensaje. Que disfrutes de la semana.

¡Feliz Semana Santa! 

Saludos,
Kasturi

**Reply by Gaga1982** [reply]
Hola Kasturi. Si. Perdón por la demora es que estoy con proyectos de trabajo en radio y espero se concreten esta semana por eso mi atraso. No tengo muy claro lo de los videos ya que en la 4ta generación de pensamiento computacional eran las otras compañeras las que se encargaban de eso
 Así que estoy tratando de editar mis videos con lo de lowfidelity

**Comment by Gaga1982** [comment]
Voy asistir a la sesión del sábado y entregar todo junto si se puede estos días. Perdón la demora se me re complico el tiempo este mes.

**Reply by KasturiSShah** [reply]
Hola @Gaga1982 

Listo, entonces nos vemos en unas horas. No te preocupes, entiendo completamente. Estaré muy atenta a este hilo para revisar las etapas cuando las puedas subir. Una cosita -- la etapa 5 es muy corta y no necesita que la etapa 4 sea completada. Así que una opción sería pasar ~ 30 minutos completando la etapa 5, entregarla, y luego pasar a la etapa 4 que necesita un poquito más trabajo. Depende de ti como quieres avanzar, solo pongo la sugerencia aquí por si te sirve.

Cualquier pregunta / duda, aquí estoy.

Mucha suerte, mucho éxito,
Kasturi

**Comment by KasturiSShah** [comment]
Hola @Gaga1982 

Espero estés muy bien. Te escribo para preguntarte ¿si quieres presentar el sábado durante el cierre del curso? Te agendaré a presentar el sábado con gusto, si quieres.

Anteriormente te asignamos a presentar ayer, miércoles. Ya que no te vi en la sesión, quisiera ofrecer otra oportunidad 😄 

Quedo atenta, lindo día,
Kasturi
